arch/tools/cli/config_generator.py

```diff
@@ -49,7 +49,9 @@ def validate_and_render_schema():
 
     if "prompt_targets" in config_yaml:
         for prompt_target in config_yaml["prompt_targets"]:
-            name = prompt_target.get("endpoint", {}).get("name", "")
+            name = prompt_target.get("endpoint", {}).get("name", None)
```

Comment:
Can you add a comments section on what this does? Does this remove the dependency on having an endpoint for a prompt_target?

Reply:
this was unrelated code - removed it

Reply:
let me think a bit more over it